### PR TITLE
Added `ShivenduKumar.json` file 

### PR DESCRIPTION
**Pull request by Shivendu-Kumar**

### CODE REVIEW

File: data/users/ShivenduKumar.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Shivendu Kumar",
+  "bio": "Android Dev",
+  "image": "https://avatars.githubusercontent.com/u/86725834?v=4",
+  "categories": [
+    {
+      "slug": "app-development",
```
Comment (codewithvoid) [inline data/users/ShivenduKumar.json:7]: this should be mobile-development based as per categories listed here
https://github.com/codewithvoid/AskDevs/blob/main/public/categories.json

File: data/users/ShivenduKumar.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (codewithvoid) [inline data/users/ShivenduKumar.json:1]: file name should be <github_username>.json, so rename it to `Shivendu-Kumar.json`

**Review by RakeshSangem — APPROVED** [review]
Looks good to me👍

**Review by codewithvoid — APPROVED** [review]

**Comment by codewithvoid** [comment]
Thanks Shivendu :)

**Comment by Shivendu-Kumar** [comment]
My pleasure Void 🤗

**Pull request merged**